nimbus/src/http_client.rs

```diff
@@ -199,36 +169,33 @@ mod tests {
         let http_client = Client::new("messaging-experiments", Some(config)).unwrap();
         let resp = http_client.get_experiments().unwrap();
         m.expect(1).assert();
-        assert_eq!(resp.len(), 2);
+        assert_eq!(resp.len(), 1);
         let exp = &resp[0];
         assert_eq!(exp.clone(), Experiment {
-            id: "ABOUTWELCOME-PULL-FACTOR-REINFORCEMENT-76-RELEASE".to_string(),
-            enabled: true,
-            targeting: Some("(firefoxVersion >= 76 && firefoxVersion < 77 && channel == 'release') || (locale == 'en-US' && [userId, \"aboutwelcome-pull-factor-reinforcement-76-release\"]|bucketSample(0, 2000, 10000) && (!('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue) || 'bug-1637316-message-aboutwelcome-pull-factor-reinforcement-76-rel-release-76-77' in activeExperiments))".to_string()),
-            arguments: ExperimentArguments {
-                    slug: "bug-1637316-message-aboutwelcome-pull-factor-reinforcement-76-rel-release-76-77".to_string(),
-                    user_facing_name: "About:Welcome Pull Factor Reinforcement".to_string(),
-                    user_facing_description: "4 branch experiment different variants of about:welcome with a goal of testing new experiment framework and get insights on whether reinforcing pull-factors improves retention. Test deployment of multiple branches using new experiment framework".to_string(),
-                    is_enrollment_paused: true,
-                    active: true,
-                    bucket_config: BucketConfig {
-                        randomization_unit: RandomizationUnit::NormandyId,
-                        namespace: "bug-1637316-message-aboutwelcome-pull-factor-reinforcement-76-rel-release-76-77".to_string(),
-                        start: 0,
-                        count: 2000,
-                        total: 10000
-                    },
-                    start_date: serde_json::from_str("\"2020-06-17T23:20:47.230Z\"").unwrap(),
-                    end_date: None,
-                    proposed_duration: Some(28),
-                    proposed_enrollment: 7,
-                    reference_branch: Some("control".to_string()),
-                    features: vec![],
-                    branches: vec![
-                        Branch { slug: "control".to_string(), ratio: 1, feature: FeatureConfig { feature_id: Group::Cfr, enabled: true, value: None } },
-                        Branch { slug: "treatment-variation-b".to_string(), ratio: 1, feature: FeatureConfig { feature_id: Group::Cfr, enabled: true, value: None } }
-                    ]
-                },
+            slug: "mobile-a-a-example".to_string(),
```

Comment:
This matches the example mobile experiment added in https://github.com/mozilla/nimbus-shared/pull/123